.gitignore

```diff
@@ -21,6 +21,9 @@ log.txt
 # Pycharm stuff
 .idea
 
+# VSCode stuff
+.vscode
+
```

Comment:
This isn't really part of this PR, is it?
 
```suggestion
```

Reply:
Just updated in this PR itself as I was using VSCode

Reply:
I'm going to take this out of this PR and put it in as a separate commit to `master`.